utils.py

```diff
@@ -48,6 +53,18 @@
 SECONDS_IN_HOUR = 60 * 60
 SECONDS_IN_MINUTE = 60
 
+T = TypeVar('T')
```

Comment:
Where is this used? (Are we going to use this?)

Reply:
I had a similar conversation with Vojta, please see the [thread](https://github.com/oppia/oppia/pull/13060#discussion_r651736365). And yes they will be used. Thanks!